mne/datasets/eegbci/eegbci.py

```diff
@@ -149,20 +158,39 @@ def load_data(
     For example, one could do::
 
         >>> from mne.datasets import eegbci
-        >>> eegbci.load_data(1, [6, 10, 14], "~/datasets") # doctest:+SKIP
+        >>> eegbci.load_data([1, 2], [6, 10, 14], "~/datasets") # doctest:+SKIP
 
-    This would download runs 6, 10, and 14 (hand/foot motor imagery) runs from subject 1
-    in the EEGBCI dataset to "~/datasets" and prompt the user to store this path in the
-    config (if it does not already exist).
+    This would download runs 6, 10, and 14 (hand/foot motor imagery) runs from subjects
+    1 and 2 in the EEGBCI dataset to "~/datasets" and prompt the user to store this path
+    in the config (if it does not already exist).
 
     References
     ----------
     .. footbibliography::
     """
     import pooch
 
+    # XXX: Remove this with mne 1.9 ↓↓↓
+    # Also remove the subject parameter at that point.
+    # Also remove the `None` default for subjects and runs params at that point.
+    if subject is not None:
+        subjects = subject
+        warn(
+            "The ``subject`` parameter is deprecated and will be removed in version "
+            "1.9. Use the ``subjects`` parameter (note the `s`) to suppress this "
+            "warning.",
+            DeprecationWarning,
```

Comment:
We changed the class we use for deprecations to FutureWarning
```suggestion
            FutureWarning,
```